downloadDirectory/pom.xml

```diff
@@ -104,6 +106,11 @@
 			<artifactId>json-simple</artifactId>
 			<version>1.1.1</version>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-compress</artifactId>
+			<version>1.18</version>
```

Comment:
I like the extraction of the versions to properties!  I know it only appears once, but could this one be put in that section too so all dependency version are together?

Reply:
That would suggest ALL version numbers should be replaced with properties, which I think would create needless bloat in the pom file, because there would be quite a few property values for dependency versions that are only used once within the file.

Reply:
Ah okay, nvm.  I didn't see the whole POM file when I made the comment and didn't realize there were as many dependencies as there are.